### PR TITLE
HSEARCH-2637 JSR-352 Review the use of cacheable and StatelessSession 

### PR DESCRIPTION
**Pull request by yrodiere**
https://hibernate.atlassian.net/browse/HSEARCH-2637

### CODE REVIEW

**Comment by mincong-h** [comment]
Hi @yrodiere, should I take a look your PR? If yes, I'll do it this weekend.

**Comment by yrodiere** [comment]
@mincong-h No no no, enjoy your holidays, this can wait ;) I just sent the PR so we don't forget about it.

File: documentation/src/main/asciidoc/manual-index.asciidoc [hunk 1]
```diff
@@ -450,12 +450,14 @@ takes place after the purge operation and before indexing.
 |Specify whether the mass indexer should be optimized at the end of the job. This operation takes
 place after indexing.
 
-|`cacheable`
-|`cacheable(boolean)`
+|`cacheMode`
+|`cacheMode(CacheMode)`
 |Optional
 |False
```
Comment (mincong-h) [inline documentation/src/main/asciidoc/manual-index.asciidoc]: The default value is now changed to `IGNORE`.

**Review by mincong-h — APPROVED** [review]
Looks good, thanks!

**Comment by yrodiere** [comment]
Fixed the doc and merged, thanks!

**Pull request merged**